docs/api-reference/mapbox/mapbox-overlay.md

```diff
@@ -133,7 +133,7 @@ new MapboxOverlay(props);
 
 The constructor additionally accepts the following option:
 
-- `interleaved` (Boolean) - If `false`, a dedicated deck.gl canvas is added on top of the base map. If `true`, deck.gl layers are inserted into mapbox-gl's layer stack, and share the same WebGLRenderingContext as the base map. Default is `false`.
+- `interleaved` (Boolean) - If `false`, a dedicated deck.gl canvas is added on top of the base map. If `true`, deck.gl layers are inserted into mapbox-gl's layer stack, and share the same WebGL2RenderingContext as the base map. Default is `false`. Interleaving with WebGL1 is not supported. See [compatibility](./overview#compatibility).
```

Comment:
Note to maintainers: Would it be more aligned with other behavior to error out when trying to interleave with WebGL1, or to gracefully ignore the interleave property and revert to the default overlay mode?

cc: @felixpalmer @Pessimistress

Reply:
It looks like [`MapboxLayer`](https://github.com/visgl/deck.gl/blame/ff0cc12b50b852a265a0c08a91aa1946d111ccdb/modules/mapbox/src/mapbox-layer.ts#L37-L40) implements a graceful solution and logs a warning. 

We probably need to do something similar in [MapboxOverlay](https://github.com/visgl/deck.gl/blob/ff0cc12b50b852a265a0c08a91aa1946d111ccdb/modules/mapbox/src/mapbox-overlay.ts#L94-L95). @felixpalmer, does that seem right?